src/containers/forms/PlanForm/tests/index.test.tsx

```diff
@@ -0,0 +1,229 @@
+    expect(toJson(wrapper.find('#interventionType select'))).toMatchSnapshot(
+      'interventionType field'
+    );
+    expect(toJson(wrapper.find('#fiStatus select'))).toMatchSnapshot('fiStatus field');
+    expect(toJson(wrapper.find('#fiReason select'))).toMatchSnapshot('fiReason field');
+    // caseNum and opensrpEventId are not rendered by default
+    expect(wrapper.find('#caseNum').length).toEqual(0);
+    expect(wrapper.find('#opensrpEventId').length).toEqual(0);
+    expect(toJson(wrapper.find({ for: 'title' }))).toMatchSnapshot('title label');
+    expect(toJson(wrapper.find('#title input'))).toMatchSnapshot('title field');
+    expect(wrapper.find({ for: 'name' }).length).toEqual(0);
+    expect(toJson(wrapper.find('#name input'))).toMatchSnapshot('name field');
+    expect(wrapper.find({ for: 'identifier' }).length).toEqual(0);
+    expect(toJson(wrapper.find('#identifier input'))).toMatchSnapshot('identifier field');
+    expect(wrapper.find({ for: 'version' }).length).toEqual(0);
+    expect(toJson(wrapper.find('#version input'))).toMatchSnapshot('version field');
+    expect(toJson(wrapper.find({ for: 'status' }))).toMatchSnapshot('status label');
+    expect(toJson(wrapper.find('#status select'))).toMatchSnapshot('status field');
+    expect(toJson(wrapper.find({ for: 'start' }))).toMatchSnapshot('start label');
+    expect(toJson(wrapper.find('#start input'))).toMatchSnapshot('start field');
+    expect(toJson(wrapper.find({ for: 'end' }))).toMatchSnapshot('end label');
+    expect(toJson(wrapper.find('#end input'))).toMatchSnapshot('end field');
+    expect(wrapper.find({ for: 'date' }).length).toEqual(0);
+    expect(toJson(wrapper.find('#date input'))).toMatchSnapshot('date field');
+    expect(toJson(wrapper.find('#submit-button button'))).toMatchSnapshot('submit button');
+
+    // if you set fiReason to case triggered then caseNum and opensrpEventId are now rendered
+    wrapper
+      .find('#fiReason select')
+      .simulate('change', { target: { value: 'Case Triggered', name: 'fiReason' } });
+    expect(toJson(wrapper.find({ for: 'caseNum' }))).toMatchSnapshot('caseNum label');
+    expect(toJson(wrapper.find('#caseNum input'))).toMatchSnapshot('caseNum field');
+    expect(wrapper.find({ for: 'opensrpEventId' }).length).toEqual(0);
+    expect(toJson(wrapper.find('#opensrpEventId input'))).toMatchSnapshot('opensrpEventId field');
+
+    // set it back
+    wrapper
+      .find('#fiReason select')
+      .simulate('change', { target: { value: 'Routine', name: 'fiReason' } });
+    expect(wrapper.find('#caseNum').length).toEqual(0);
+    expect(wrapper.find('#opensrpEventId').length).toEqual(0);
+
+    wrapper.unmount();
+  });
+
+  it('renders activity fields correctly', () => {
+    function checkActivities(num: number, name: string = 'FI') {
+      // FI activities by default
+      for (let i = 0; i <= num; i++) {
+        expect(toJson(wrapper.find({ for: `activities-${i}-actionTitle` }))).toMatchSnapshot(
```

Comment:
We are breaking down the snapshot into smaller more useful pieces to avoid giant snapshots